src/js/services/lang.js

```diff
@@ -9,15 +9,15 @@ const languages = {
   en: {
     name: 'English',
     url: 'docs/',
-    id: 'globalNavLangEng',
-    domEl: getElById('globalNavLangEng'),
+    id: 'globalNavLang-en',
+    domEl: getElById('globalNavLang-en'),
```

Comment:
Taking a look at test 'should go through sidebar yml to see if all links exist in EN then JA' explains it well. Basically the yml file is split into 2 the JA version and the EN version. To avoid parsing the yml file twice and reusuing the code i changed the button name that you need to press in order to change the language. 

In hinde site I could have done each time the yml file goes into a new language then go to the new url but I figured that the button would have the url so if the URL ever changes I wouldn't have to update it in 2 places but I assumed there will always be a button to select the language just a matter of different location. 

I am up for changing but that was the logic behind the choice.

Reply:
The url for JA ends in ja but it doesn't end in /en for english. It was easier to just change the button name as it seemed to have a consistent pattern for testing in the future. You need to have the right language selected / correct language url to show the side bar for the specific language. Let me know if have any other questions but I think the test kinda explains the logic as well :)

Reply:
if I understand correctly, you are using the dynamic values of the sidebar to select the values from the javascript in this line:

```
cy.get(`#globalNavLang-${key}`).click()
```

I think this is fine to do - we should standardize our use of languages to be `ISO 639-1` (two letter-codes) when we can anyway.

Reply:
i believe that the languages is 2 letter codes 'en' and 'ja' but good call out for future languages and yes you are correct on the process

Reply:
this is a very cool way to do this, good stuff chantelle!